lib/api.js

```diff
@@ -43,7 +43,7 @@ function createLabels(labels) {
       owner,
       repo,
       name: label.name,
-      color: label.color,
+      color: label.color || 'ffffff',
```

Comment:
I think we should make this a required field instead, you would usually always want to specify the color yourself.

Reply:
Yes, I also think that's the better solution 👍 
116ce33